cg/meta/workflow/balsamic.py

```diff
@@ -278,6 +279,12 @@ def get_verified_sex(sample_data: dict) -> Sex:
             LOG.error(f"Unable to retrieve a valid sex from samples: {sample_data.keys()}")
             raise BalsamicStartError
 
+    def get_exome_argument_if_exome_sample(self, case_id: str) -> bool:
```

Comment:
```suggestion
    def is_exome_sample(self, case_id: str) -> bool:
```
For functions returning bool, prepend with `is_`